downtime/models.py

```diff
@@ -14,4 +14,4 @@ class Period(models.Model):
     objects = PeriodManager()
 
     def __str__(self):
-        return "Scheduled downtime"
\ No newline at end of file
+        return "Scheduled Downtime: %s to %s" % (self.start_time, self.end_time)
```

Comment:
Need to check for the existence of both ```self.start_time``` and ```self.end_time```.

It is possible to have downtime objects without those two traits.